index.html

```diff
@@ -14,6 +17,125 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="container">
+    <header>
+
+
+      <nav>
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+
+
+      <div class="intro">
+        <div class="button-intro">
+          <h1>Kimberly Swinton</h1>
+          <button>LET'S TALK</button>
+        </div>
+
+        <h3>Full Stack Engineer</h3>
+
+      </div>
+
+
+      <div class="Objectives">
+        <h3>Career Objectives</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+          magna aliqua. Ultricies integer quis auctor elit sed. Pulvinar proin gravida hendrerit lectus a. Et pharetra
+          pharetra massa massa ultricies. Habitant morbi tristique senectus et netus et.</p>
+      </div>
+
+    </header>
+
+    <div class="aboutme">
+      <div>
+        <h3>ABOUT ME</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+          magna aliqua. Ultricies integer quis auctor elit sed. Pulvinar proin gravida hendrerit lectus a.
+          Et pharetra pharetra massa massa ultricies. Habitant morbi tristique senectus et netus et.</p>
+      </div>
+
+      <div class="top-section">
+        <div>
```

Comment:
Whenever you use a `<div>` always give it a class. Divs without a class name, are called floating divs, and developers don't want to see that